pkg/logger/debugger_redis.go

```diff
@@ -0,0 +1,152 @@
+package logger
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/redis/go-redis/v9"
+)
+
+const (
+	debugRedisAddChannel = "add:log-debug"
+	debugRedisRmvChannel = "rmv:log-debug"
+	debugRedisPrefix     = "debug:"
+)
+
+var (
+	ErrInvalidDomainFormat = errors.New("invalid domain format")
+)
+
+// RedisDebugger is a redis based [Debugger] implementation.
+// It sends the logs directly into a redis.
```

Comment:
I don't think the debugger sends logs, it only manipulates a list of Cozy domains which are in debug mode, right?

Reply:
Correct, I fix this.